src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -465,7 +466,7 @@ void createConsumerWithNotExistingParameter(VertxTestContext context) throws Int
 
     @Test
     void receiveSimpleMessage(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
-        String topic = "receiveSimpleMessage";
+        String topic = "receiveSimpleMessage-1";
```

Comment:
why changing the name?

Reply:
I have changed it to default most likely old code... )